arms/lineararm.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import cvxpy as cp
+from .utils import Arm
+
+__all__ = ['LinearArm']
+
+
+class LinearArm(Arm):
```

Comment:
Is it better to name it "CorrelatedArm"? It is confusing considering the well-known linear bandit.